src/main/java/pro/sky/calculator/Controller.java

```diff
@@ -0,0 +1,70 @@
+package pro.sky.calculator;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/calculator")
+public class Controller {
+
+    private final ServiceImpl serviceImpl;
```

Comment:
```suggestion
    private final Service serviceImpl;
```
Всегда нужно использовать интерфейсную ссылку, чтобы использовать полиморфизм